release/platform/platform_test.go

```diff
@@ -116,3 +116,18 @@ func TestPlatformsHaveNoDuplicates(t *testing.T) {
 		assert.Equal(t, 1, dupes, "platform %v only occurs once in platforms list", p1)
 	}
 }
+
+func TestPlatformCorrectArch(t *testing.T) {
```

Comment:
I was thinking something that would actually determine the local Linux distro and check that the arch is correct. But this probably suffices.